### PR TITLE
Add package nptr

### PR DESCRIPTION
**Pull request by henryas**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -22188,5 +22188,19 @@
     "description": "create db util sources from csv",
     "license": "MIT",
     "web": "https://github.com/z-kk/csv2dbsrc"
+  },
+  {
+    "name": "nptr",
+    "url": "https://15b1acb3d21ebde45e8ab271e72fb4b75ad40143:x-oauth-basic@github.com/henryas/nptr",
```
Comment (narimiran) [inline packages.json:22194]: Why not https://github.com/henryas/nptr ?

Comment (henryas) [reply packages.json:22194]: It was autogenerated by nimble. It should be https://github.com/henryas/nptr

Comment (henryas) [reply packages.json:22194]: This is my first nimble package and I am new to this. So do let me know if there is anything else I need to do. Thank you.

Comment (xflywind) [reply packages.json:22194]: You can edit the link directly.

Comment (xflywind) [inline packages.json:22194]:
```suggestion
    "url": "https://github.com/henryas/nptr",
```

File: packages.json [hunk 2]
```diff
@@ -22188,5 +22188,19 @@
...
+    "method": "git",
+    "tags": [
+      "smart",
+      "pointer",
+      "pointers",
+      "concurrency"
+    ],
+    "description": "Nim lang smart pointers",
+    "license": "MIT",
+    "web": "https://15b1acb3d21ebde45e8ab271e72fb4b75ad40143:x-oauth-basic@github.com/henryas/nptr"
```
Comment (xflywind) [inline packages.json:22204]:
```suggestion
    "web": "https://github.com/henryas/nptr"
```

File: packages.json [hunk 3]
```diff
@@ -22188,5 +22188,19 @@
     "description": "create db util sources from csv",
     "license": "MIT",
     "web": "https://github.com/z-kk/csv2dbsrc"
+  },
+  {
+    "name": "nptr",
+    "url": "https://15b1acb3d21ebde45e8ab271e72fb4b75ad40143:x-oauth-basic@github.com/henryas/nptr",
+    "method": "git",
+    "tags": [
+      "smart",
```
Comment (xflywind) [inline packages.json:22197]:
```suggestion
```
？

Comment (Yardanico) [reply packages.json:22197]: @xflywind smart pointers

Comment (xflywind) [reply packages.json:22197]: yeah I mean tags should be `smart pointers` not `smart`?

**Pull request merged**